### PR TITLE
Start to switch to WC2

### PR DESCRIPTION
**Pull request by hayesgm**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -11,13 +11,18 @@ inputs:
     default: 8585
   requested_network:
     description: "Requested network to connect to"
+    required: true
+    default: 1
+  wallet_connect_project_id:
+    description: "Your WalletConnect project id [defaults to secret WALLET_CONNECT_PROJECT_ID]"
+    required: true
```
Comment (alexandfox) [inline action.yml:18]: qq - do we need `required: true` if we set default here? just one less thing need to update / add to current seacrest implementations if not

Comment (alexandfox) [reply action.yml:18]: ah nvm, i think this is referring to the step. maybe just remove duplicate line then

File: src/connector.mjs [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+  'goerli': 5
 };
 
-export async function getWalletConnector(connectOpts={}) {
+export async function getWalletConnector(walletConnectProjectId, relayUrl, requestedNetwork, connectOpts={}) {
   const opts = {
-    requestedNetwork: null, // take any
     reshowDelay: null,
-    uriTimeout: 60000,
+    uriTimeout: 120000,
```
Comment (alexandfox) [inline src/connector.mjs:12]: yay 😎

**Review by alexandfox — APPROVED** [review]
thanks! left one comment

may be helpful to update readme as well just to specify that need to pass both `WALLET_CONNECT_PROJECT_ID` and `REQUESTED_NETWORK` now

**Review by cwang25 — APPROVED** [review]

File: src/connector.mjs [hunk 3]
```diff
@@ -1,22 +1,28 @@
-import NodeWalletConnect from "@walletconnect/node";
+import SignClient from '@walletconnect/sign-client'
 import qrcode from "qrcode-terminal";
 
 const networks = {
-  1: "mainnet",
-  5: "goerli",
-  42: "kovan",
+  'mainnet': 1,
+  'goerli': 5
```
Comment (cwang25) [inline src/connector.mjs:6]: Should we add more networks here beyond just `mainnet` and `goerli` ? 
Maybe arbitrum/arbitrum-goerli?

**Pull request merged**